deploy/systemd/granulate-gprofiler.service.template

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+[Unit]
+Description=Granulate gProfiler Agent
+Documentation=https://profiler.granulate.io/
+After=network.target
+
+[Service]
+Type=forking
+PIDFile=/var/gprofiler.pid
+User=root
+Environment=GPROFILER_SERVICE=<insert-service-name>
+Environment=GPROFILER_TOKEN=<insert-token>
+Environment=GPROFILER_VERSION=<insert-version>
+WorkingDirectory=/opt/granulate/gprofiler
+ExecStartPre=/bin/bash -c 'if [ -z "${GPROFILER_TOKEN}" ]; then echo "missing GPROFILER_TOKEN!"; exit 1; fi'
+ExecStartPre=/bin/bash -c 'if [ -z "${GPROFILER_SERVICE}" ]; then echo "missing GPROFILER_SERVICE!"; exit 1; fi'
+ExecStartPre=/bin/bash -c 'wget --no-verbose https://github.com/Granulate/gprofiler/releases/${GPROFILER_VERSION}/download/gprofiler_$(uname -m) -O gprofiler && chmod +x gprofiler'
```

Comment:
This won't work for both "latest" and "version". The URL for version is releases/download/version/gprofiler.

Don't need to support it, anyway

Reply:
Only latest